src/themes/default-theme/colors.js

```diff
@@ -1,7 +1,7 @@
 import { darken, lighten } from 'polished';
 
 const colorsToShade= {
-	primary: 'teal',
+	primary: '#008080', // teal
```

Comment:
The palette colors in the default theme have up until now been using CSS color names only. It would be nice if we could keep it that way.

Reply:
As far as I can see, there are three more colors in this theme which use color code.
I'm gonna try to add ability to get color code from color name to `stringToRGB` and then get rid of changing themes